app/src/main/java/by/hometrainng/androidcalculator/utils/OperationalList.kt

```diff
@@ -0,0 +1,15 @@
+package by.hometrainng.androidcalculator.utils
+
+object OperationalList { // TODO реализовать передачу через bundle
+
+    private var list = mutableListOf<String>()
+
+    fun addElement(item: String) {
+        list.add(item)
+    }
+
+    fun getList() : List<String> {
+        return list
+    }
```

Comment:
чтобы отдать новую коллекцию которую нельзя будет изменить
```suggestion
    fun getList() : List<String> {
        return list.toList()
    }
```